CHANGELOG.md

```diff
@@ -44,8 +44,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#2580](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2580))
 - Populate `{method}` as `HTTP` on `_OTHER` methods from scope for `asgi` middleware
   ([#2610](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2610))
-  - Populate `{method}` as `HTTP` on `_OTHER` methods from scope for `fastapi` middleware
-  ([#2682](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2682))
+- Populate `{method}` as `HTTP` on `_OTHER` methods from scope for `fastapi` middleware
+([#2682](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2682))
+- `urllib3`: add `method` and `url` parameters to the request hook
```

Comment:
Nit: If you simply say "add" it wouldn't be a breaking change. I would just say "refactor to include `method` and `url`.